bga_database/static/bootstrap/scss/utilities/_position.scss

```diff
@@ -35,3 +35,11 @@ $positions: static, relative, absolute, fixed, sticky;
     z-index: $zindex-sticky;
   }
 }
+
+.sticky-top-button {
+  @supports (position: sticky) {
+    position: sticky;
+    top: 55px;
+    z-index: $zindex-sticky;
+  }
+} 
```

Comment:
The `static/bootstrap/*` Sass files define a custom Bootstrap build, which the `bootstrap-overrides.css` file then, as you might guess, overrides. (Docs would be too convenient, but I took and documented a similar approach over in CA FWD: https://github.com/datamade/california-dream-index#compiling-sass-to-css) 

So, you only need to define this new custom class in `bootstrap-overrides.css`.

Reply:
Actually, the most appropriate place would probably be [`custom.css`](https://github.com/datamade/bga-payroll/blob/master/bga_database/static/css/custom.css).

Reply:
that makes sense x 2. changed here d41957b. @fgregg is this ready to merge otherwise?